src/components/FromDestinationForm.js

```diff
@@ -1,40 +1,62 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-
+import { FaTimes } from 'react-icons/fa';
```

Comment:
although this is irrelevant change, clearing the form should have been a separate issue. But nice work. Will add the issue and link it to this PR